app/actions/goals-list.js

```diff
@@ -0,0 +1,6 @@
+import { createAction } from 'redux-actions';
+
+// define actions here
+const stepAddGoal = createAction('STEP_ADD_GOAL');
```

Comment:
So you should have a file called action_types.js that looks like this:
```js
export const STEP_ADD_GOAL = 'STEP_ADD_GOAL'
```
I know it seems like a pain, but it avoids typos.

So here you would have
```js
import { STEP_ADD_GOAL } from '../action_types.js'

const stepAddGoal = createAction(STEP_ADD_GOAL);
```